en/docs/get-started/quick-start-guide.md

```diff
@@ -431,7 +425,18 @@ To verify whether the reading list is personalized for each user, you can log in
 
 Currently, the updates to the `readinglist` REST API are saved only as long as the deployment status of the `readingList` API remains **Active**. If you undeploy and redeploy the `readingList` API, Choreo deploys it in a new container. Therefore you cannot retrieve the reading list with the updates you made after the last deployment. To persist the data even after undeploying and redeploying the REST API, let's connect it to a MySQL database. Then let's create and publish a new version of the REST API with the ability to persist the data to the MySQL database.
 
-### Step 3.1: Create a new version of the REST API
+### Step 3.1: Provision a database
+
+To persist your updates to the reading list, provision a MySQL database on a cloud-based platform. For example, see [AWS Documentation - Provision a database](https://aws.amazon.com/getting-started/hands-on/inventory-system-for-gaming-app-with-amazon-aurora-serverless/3/).
+
+To allow Choreo to interact with the database, add the following IPs to the allow-list of the provisioned database:
```

Comment:
Should we change allow-list to allowlist based on the usage suggested in the Google style guide [1]?
[1] https://developers.google.com/style/word-list#allowlist